agentdock-core/src/storage/adapters/mongodb/connection.ts

```
@@ -0,0 +1,99 @@
+  protected async closeConnection(): Promise<void> {
+    if (this.connection) {
+      await this.connection.client.close();
+    }
+  }
+
+  /**
+   * Check if connected
+   */
+  isConnected(): boolean {
+    return !!(
+      this.connection && this.connection.client.topology?.isConnected?.()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)